web/src/app/services/ServiceForm.tsx

```diff
@@ -1,13 +1,29 @@
 import React from 'react'
-import p from 'prop-types'
 import Grid from '@material-ui/core/Grid'
 import TextField from '@material-ui/core/TextField'
 import { EscalationPolicySelect } from '../selection/EscalationPolicySelect'
 import { FormContainer, FormField } from '../forms'
 
-export default function ServiceForm(props) {
-  const { epRequired, ...containerProps } = props
+interface ServiceFormProps {
+  value: {
+    name: string
+    description: string
+    escalationPolicyID: string
+
+    errors: {
+      field: ['name', 'description', 'escalationPolicyID']
+      message: string
+    }
```

Comment:
Errors should be an array

Reply:
Changed errors to an array. Also, changed `field` to a union between the values:
```
errors: {
    field: 'name' | 'description' | 'escalationPolicyID'
    message: string
  }[]
```